tests/acceptance/editor-copy-paste-test.js

```diff
@@ -407,7 +407,7 @@ test('paste with shift key pastes plain text', (assert) => {
   Helpers.dom.triggerCopyEvent(editor);
   editor.selectRange(editor.post.tailPosition());
 
-  Helpers.dom.triggerKeyEvent(editor, 'keydown', { keyCode: Keycodes.SHIFT });
+  Helpers.dom.triggerKeyEvent(editor, 'keydown', { keyCode: Keycodes.SHIFT, shiftKey: true });
```

Comment:
This will update the modifier map on the event manager:

```js
Helpers.dom.triggerKeyEvent(editor, 'keydown', { keyCode: Keycodes.SHIFT });
```

But I don't believe this will:

```js
Helpers.dom.triggerKeyEvent(editor, 'keydown', { shiftKey: true });
```

Why add the `shiftKey` value?

Reply:
Oops, that is a hangover from an earlier iteration I had where the modifier map was updated by looking at the `key.shiftKey` value on every keyup/keydown event.